build.go

```diff
@@ -114,14 +163,60 @@ func Build(entries EntryResolver,
 		logger.Action("Completed in %s", duration.Round(time.Millisecond))
 		logger.Break()
 
-		err = environment.Configure(phpLayer)
+		logger.Debug.Subprocess("Finding PHP extensions directory")
+		extensionsDir, err := files.FindExtensions(phpLayer.Path)
+		if err != nil {
+			return packit.BuildResult{}, err
+		}
+		logger.Debug.Break()
+
+		libDir := filepath.Join("lib")
```

Comment:
Is there a reason we do a `filepath.Join()` here instead of just `libDir := "lib"`?

Reply:
No reason. Good catch.